### PR TITLE
FSMA-104 Add BVN to create and edit client

### PR DESCRIPTION
**Pull request by Deepika1095**
https://fiterio.atlassian.net/browse/FSMA-104

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Review by josephNak — APPROVED** [review]

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/FSMA-104_add_bvn_in_m_client.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="deepika@fiter.io" id="FSMA-104_add_bvn_in_m_client">
+
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="m_client" columnName="bvn"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="m_client">
+            <column name="bvn" type="BIGINT"/>
```
Comment (oluexpert99) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/FSMA-104_add_bvn_in_m_client.xml:31]: should be a varchar , bvn is like 11 dgits for now but can always change

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/client/data/ClientData.java [hunk 2]
```diff
@@ -361,13 +365,14 @@ public static ClientData template(final Long officeId, final LocalDate joinedDat
         final Boolean isStaff = false;
         final ClientNonPersonData clientNonPersonDetails = null;
         final Set<ClientCollateralManagementData> clientCollateralManagements = null;
+        final Long bvn = null;
```
Comment (oluexpert99) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/client/data/ClientData.java:368]: better as String bvn

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/FSMA-104_add_bvn_in_m_client.xml (hunk 1 above)
Comment (Deepika1095) [reply fineract-provider/src/main/resources/db/changelog/tenant/parts/FSMA-104_add_bvn_in_m_client.xml:31]: Specially mention in requirement, it should allow integers only, and and length is 20 so working with more than 10.

**Review by oluexpert99 — APPROVED** [review]
LGTM

**Pull request merged**